.clang-tidy

```diff
@@ -1,6 +1,13 @@
 Checks: '-*,readability-identifier-naming'
 CheckOptions:
-- key:             readability-identifier-naming.PrivateMemberPrefix
+- key:             readability-identifier-naming.MemberPrefix
+  value:           'm_'
```

Comment:
This includes `struct` as well but shouldn't.

Reply:
> All member variables start with m_, all function parameters start with a_, all class names start with c.

Do you agree @madmaxoft ? Then I would change the sentence to

> All class member variables start with m_, all function parameters start with a_, all class names start with c.